CXL/ruleset.xml

```diff
@@ -126,9 +130,10 @@
         <exclude name="SlevomatCodingStandard.Functions.DisallowArrowFunction.DisallowedArrowFunction"/>
         <exclude name="SlevomatCodingStandard.Functions.RequireMultiLineCall"/>
         <exclude name="SlevomatCodingStandard.Functions.TrailingCommaInCall"/>
+        <exclude name="SlevomatCodingStandard.Namespaces.FullyQualifiedGlobalConstants.NonFullyQualified"/>
         <exclude name="SlevomatCodingStandard.Namespaces.FullyQualifiedGlobalFunctions.NonFullyQualified"/>
         <exclude name="SlevomatCodingStandard.Namespaces.UseOnlyWhitelistedNamespaces"/>
-        <exclude name="SlevomatCodingStandard.Classes.ParentCallSpacing.IncorrectLinesCountAfterLastControlStructure"/>
+        <exclude name="SlevomatCodingStandard.PHP.RequireNowdoc.RequiredNowdoc"/>
```

Comment:
It's screaming for `heredoc` syntax which we use in quite many places. 

As [PHP docs mention](https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc):

> Nowdocs are to single-quoted strings what heredocs are to double-quoted strings. A nowdoc is specified similarly to a heredoc, but no parsing is done inside a nowdoc.

So basically:
* `heredoc` === `"$var"`
* `nowdoc` === `'$var'`

With small difference in syntax:
* `heredoc` -> `<<<HTML`
* `nowdoc` -> `<<<'HTML'`

I don't think we need such "improvement" with current code base.